src/cli-app/NOTICE

```diff
@@ -0,0 +1,104 @@
+Copyright 2010, JA-SIG, Inc.
```

Comment:
This is a downloadable application that requires a NOTICE, this is the only one that contains bundled software.